baitap-submit/nguyen_duy2005/02-llm-api-params/bai2_thu3.py

```diff
@@ -0,0 +1,58 @@
+import os
+
+import requests
+from bs4 import BeautifulSoup
+from openai import OpenAI
+
+
+def take_info_fromhtml(url):
+    response = requests.get(url)
+    soup = BeautifulSoup(response.content, "html.parser")
+    paragraphs = [p.text for p in soup.find_all("p")]
+    new_paragraphs = "\n".join(paragraphs[:15])
+    return str(new_paragraphs)
+
+
+def create_prompt(user_input):
+    global memory_ai
```

Comment:
như trên nha, mình nên hạn chế dùng global variable, như cái history message này hoàn toàn có thể làm thành tham số để truyền vào function